utils/dataset_manifest/create.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+import argparse
+import os
+import sys
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--type', type=str, choices=('video', 'images'), help='Type of datset data', required=True)
+    parser.add_argument('manifest_directory',type=str, help='Directory where the manifest file will be saved')
```

Comment:
Why not `--output-file,-o file`?

Reply:
Due to the fact that the manifest file name is predefined, the current solution avoids unnecessary actions, such as a different analysis of uploaded files, etc.

Reply:
OK. Agree. Let's make the current directory as the default value.